client/js/lounge.js

```diff
@@ -80,6 +80,10 @@ $(function() {
 		});
 	});
 
+	socket.on("authorized", function() {
+		$("#loading-page-message").text("Authorized, loading messages…");
```

Comment:
On slow networks it might take a while to receive the `init` packet containing all the network/channel/message data.